missed_events_replayer/missed_events_replayer/archive.py

```diff
@@ -0,0 +1,75 @@
+import time
+
+import boto3
+
+from helpers import console_print, get_logger
+
+LOGGER = get_logger()
+DATE_RANGE_FORMAT = '%Y-%m-%d'
+MAX_WAIT_SECONDS = 600
+client = boto3.client('ecs')
+
+
+def get_matomo_container_instance_arn():
+    all_container_instance_arns = client.list_container_instances(cluster='platform-web')['containerInstanceArns']
+
+    for arn in all_container_instance_arns:
+        task_arns = client.list_tasks(cluster='platform-web', containerInstance=arn)['taskArns']
+        task_defs = client.describe_tasks(cluster='platform-web', tasks=task_arns)['tasks']
+        for task_def in task_defs:
+            if any(container['name'] == 'matomo' for container in task_def['containers']):
+                return arn
+
+    LOGGER.error(f"No container instances found running Matomo. Instance IDs: f{all_container_instance_arns}")
+    exit(1)
```

Comment:
This is really cool, and could serve as an example for when we (inevitably) need to do something similar in future. I think we should have a recipes section of the wiki or team manual with short snippets like this.

Reply:
Yeah that's a good idea. For those times when you know you've seen something done somewhere but can't remember where it was!